### ISSUE TITLE
Ability to use packet filtering

### ISSUE THREAD

**Issue by mabra** [vergoh/vnstat#268]
It was sad to see, that the is no packet filtering in vnstat.
So it became useless for me especially in the scenarios, where I have
to pay for traffic, but not be able to control it properly.
In ISPs/hostings scenarios, there is often interface traffic, which does not represent your traffic going over the gateway, like filesharing for my "neighbour server" (which has the highest volume!) in the same LAN and espcially in that situations, you are usually getting only one interface (cheaper for the provider ....).
There are some other network tools around, which show, that implementing tcp-dump alike filter (this is probably pcap) should not be witch-work.
Just my two pence and my hope for the future!

**Comment by vergoh** [comment]
The second sentence of the README.md says

> This means that vnStat won't actually be sniffing any traffic and also ensures light use of system resources regardless of network traffic rate.

so it shouldn't come as a surprise vnStat doesn't do packet filtering. That's also one of the key features as the system load caused by vnStat will stay exactly the same regardless of the amount of network traffic. That wouldn't be true if packet filtering would be used. As you've found, there are already other programs doing similar things with packet filtering resulting in there not being much reason developing vnStat towards that direction when, on the other hand, rather few programs not doing packet filtering exist.

**Comment by mabra** [comment]
> The second sentence of the README.md says
> .....
> .... As you've found, there are already other programs doing similar things with packet filtering
>

NO, I wrote that there are other programs which do packet-filtering, as an example, that it is not too complicatet. No other programs like vnStat exists, whicd do filtering.
This was the reason to tag it a feature request ;)
